code/modules/unit_tests/turf_icons.dm

```diff
@@ -9,7 +9,7 @@
 		var/icon_file = initial(turf_path.icon)
 		if(isnull(icon_state) || isnull(icon_file))
 			continue
```

Comment:
```suggestion
```
`icon_exists()` checks for this already

Reply:
I'd rather not apply this. This change causes a stack trace if either is null, whereas previously it treated a null `icon_state` or `icon_file` as "we should skip this tile".